tests/conftest.py

```diff
@@ -97,3 +106,44 @@ def rewrite_data_files():
     A test exists in CI to ensure that this flag is set to False on commit.
     """
     return False
+
+
+def _wait_until_webpack_ready(max_wait_sec: int = 60):
```

Comment:
@rabstejnek changed slightly here to wait intelligently rather than sleeping; hopefully this means we'll get less random pipeline failures, though w/ selenium we'll surely get some....